src/components/GameCard.js

```diff
@@ -1,68 +1,132 @@
+ 
+    const [isSetToDelete, setIsSetToDelete] = useState(false)
+
+    return(
+        <Card>
+            {!isSetToDelete && (
+                <SavedGameContent>
+                    <Location>{location}</Location>
+                    <Date>{date}</Date>
+                    <PlayerWrapper>
+                        <Headline>Player(s)</Headline>
+                        <span>{players}</span>
+                    </PlayerWrapper>
+                    <WinnerWrapper>
+                        <Headline>Winner(s)</Headline>
+                        <span>{winner}</span>
+                    </WinnerWrapper>
+                    <ShotsWrapper>
+                        <Headline>Total Shots</Headline>
+                        <span>{shots}</span>
+                    </ShotsWrapper>
+                    <ButtonDeleteFirst onClick={() => setIsSetToDelete(true)}><BinIcon/></ButtonDeleteFirst>
+                </SavedGameContent>
+            )}
+            {isSetToDelete && (
+                <DeleteField>
+                    <span>Do you want to delete this gamecard?</span>
+                    <ButtonDeleteSecond onClick={() => onDelete(id)}><BinIcon/>Delete</ButtonDeleteSecond>
+                    <ButtonCancel onClick={() => setIsSetToDelete(false)}> X Cancel</ButtonCancel>
+                </DeleteField>
+            )}
+        </Card>
+    )
 }
 
 const Card = styled.section`
     padding: 10px;
-    display: grid;
-    grid-template-rows: repeat(3, 1fr);
-    grid-template-columns: repeat(3, 1fr);
-    grid-gap: 10px;
-    justify-items: center;
     box-shadow: 0 0 10px var(--primary-medium);
     border-radius: 10px;
     background-color: var(--primary-medium);
     font-size: 0.9rem;
-    text-align: center;
+`
+const SavedGameContent = styled.div`
+    display: grid;
+    grid-template-rows: repeat(4, auto);
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 10px;
+    justify-items: left;
+`
+const Date = styled.div`
+    grid-column: 1;
+    grid-row-start: 1;
 `
 const Location = styled.div`
     grid-column: 1 / 3;
-    justify-self: start;
+    grid-row-start: 2; 
+    text-transform: uppercase;
 `
-const Date = styled.div`
-    justify-self: end;
+const PlayerWrapper = styled.div`
+    grid-column: 1/4;
+    grid-row-start: 3;
+`
+const WinnerWrapper = styled.div`
+    grid-column: 1;
+    grid-row-start: 4;
+`
+const ShotsWrapper = styled.div`
+    grid-column: 2;
+    grid-row-start: 4;
 `
 const Headline = styled.h4`
     margin: 0;
-    place-items: center;
     font-weight: 550;
 `
-const DataPlayers = styled.span`
-    grid-column: 1 / 2;
-    grid-row: 3 / 4;
+const DeleteField = styled.div`
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 10px 20px;
     place-items: center;
+    
+    span{
+        grid-column: 1/3;
+        text-align: center;
+        color: var(--secondary-dark);
+        font-weight: 800; 
+    }
```

Comment:
Maybe you wouldn't need to use a grid here. A flexbox to align the two buttons would also have been an option. But then again, it works just fine like this :)

Reply:
My idea was to stretch the text over two columns and to have the buttons placed underneath the text in separate columns.